### PR TITLE
Feat/option client id

### PR DESCRIPTION
**Pull request by rizalfadlila**
hi @mingrammer, please kindly review my pr.
`clientID` used to replace value `id` as consumer group client id. 

Thanks

### CODE REVIEW

File: kinesumer.go [hunk 1]
```diff
@@ -35,8 +35,9 @@ const (
 
 // Config defines configs for the Kinesumer client.
 type Config struct {
-	App    string // Application name.
-	Region string // Region name. (optional)
+	App      string // Application name.
+	Region   string // Region name. (optional)
+	ClientID string // consumer group client id. (optional)
```
Comment (mingrammer) [inline kinesumer.go:40]: Please use title case for comment: `Consumer group ...`

Comment (rizalfadlila) [reply kinesumer.go:40]: oke noted

**Review by mingrammer — APPROVED** [review]
LGTM

**Comment by mingrammer** [comment]
v0.3.8 is now released.

**Pull request merged**